### ISSUE TITLE
list view

### ISSUE THREAD

**Issue by acnewberry** [vinyldns/vinyldns-python#49]
Should I get a list of zones?

>>> local_client = VinylDNSClient("https://api.vinyldns.comcast.net:9443", "blah", "blah")
>>> local_client.list_zones()
<vinyldns.zone.ListZonesResponse object at 0x103ea4da0>

**Comment by pauljamescleary** [comment]
@acnewberry Yes, list_zones should return a list of _your_ zones that you have access to.  By default it maxes out at 100 zones
